card/lib/card/migration.rb

```diff
@@ -28,8 +27,12 @@ def find_unused_name base_name
       test_name
     end
 
-    def paths mig_type=type
-      Card.paths["db/migrate#{schema_suffix type}"].to_a
+    def migration_paths mig_type=type
+      Cardio.migration_paths mig_type
+    end
```

Comment:
I think this is probably a good change.  not a big deal to change it in wikirate.  I think Philipp was making that comment as a flag to me mostly, so I'll remember to update wikirate when we push this out.